setup.py

```diff
@@ -74,6 +77,19 @@ def clean_version(v):
       setup_requires=[],
       )
 
+if current_platform != "windows":
+    bash_file = "~/.bash_profile" if current_platform == "darwin" else "~/.bashrc"
```

Comment:
The two here, need cleaning

`if current_platform != "windows"`: will cover ` if current_platform == "darwin"` 

if we are not on windows that means we are on darwin(nix, osx..)